src/olcs/VectorSynchronizer.js

```diff
@@ -99,8 +100,16 @@ class VectorSynchronizer extends olcsAbstractSynchronizer {
    * @inheritDoc
    */
   createSingleLayerCounterparts(olLayerWithParents) {
-    const olLayer = olLayerWithParents.layer;
-    if (!(olLayer instanceof olLayerVector) || olLayer instanceof olLayerVectorTile) {
+    let olLayer = olLayerWithParents.layer;
+
+    if(olLayerWithParents.layer instanceof olLayerVectorImage){
+      let convertedVectorLayer = new olLayerVector({source: null});
+      convertedVectorLayer.setStyle(olLayerWithParents.layer.getStyle())
+      convertedVectorLayer.setSource(olLayerWithParents.layer.getSource())
+      olLayer = convertedVectorLayer;
```

Comment:
Please directly pass source and style as constructor parameter.
Notably, see https://openlayers.org/en/latest/apidoc/module-ol_layer_Vector-VectorLayer.html

Reply:
will fix it next week